### PR TITLE
esr: Add HVC and SMC ECs for ESR_EL2

### PR DESCRIPTION
**Pull request by vdonnefort**

### CODE REVIEW

File: src/esr/mod.rs [hunk 1]
```diff
@@ -161,6 +161,16 @@ pub fn decode(esr: u64) -> Result<Vec<FieldInfo>, DecodeError> {
...
+        0b010110 => (
+            "HVC instruction execution in AArch64 state",
+            decode_iss_hvc(iss.value)?,
+            None,
+        ),
+        0b010111 => (
+            "SMC instruction execution in AArch64 state",
+            decode_iss_hvc(iss.value)?,
+            None,
+        ),
```
Comment (qwandor) [inline src/esr/mod.rs:173]: Let's keep these sorted in numerical order.

**Review by qwandor — APPROVED** [review]

File: src/esr/mod.rs (hunk 1 above)
Comment (vdonnefort) [reply src/esr/mod.rs]: Of course, this is done!

**Comment by qwandor** [comment]
Thanks! I'll push a new release shortly.

**Comment by qwandor** [comment]
This should now be live on https://esr.arm64.dev/, and the command-line version is available from https://crates.io/crates/aarch64-esr-decoder and https://github.com/google/aarch64-esr-decoder/releases/tag/0.2.1.

**Pull request merged**